src/annotator/config/test/url-from-link-tag-test.js

```diff
@@ -0,0 +1,67 @@
+import { urlFromLinkTag } from '../url-from-link-tag';
+
+describe('annotator/config/url-from-link-tag', () => {
```

Comment:
When a test module contains only a single thing (function, class etc.) being tested and the item's name is reasonably unique, the convention is to name the `describe` block after the item. If a module contains multiple functions/classes etc. being tested, then you'd structure it as:

```js
describe('path/to/module', () => {
  describe('functionOne', () => {
     ...
  });

  describe('functionTwo', () => {
     ...
  });
});
```

This convention just makes it easier to know what a test refers to if it fails.

Reply:
okay makes sense